test/browser/events.test.js

```diff
@@ -95,6 +95,32 @@ describe('event handling', () => {
 		expect(click).to.have.been.calledOnce.and.calledWith(1);
 	});
 
+	// Uniquely named in that the base event names end with 'Capture'
+	it('should support got/lost pointer capture', () => {
+		let ongotpointercapture = sinon.spy(),
+			onlostpointercapture = sinon.spy();
+
+		render(
+			<div
+				onPointerDown={e => e.target.setPointerCapture(e.pointerId)}
```

Comment:
Struggling a bit with this test. For some reason, this always results in the following error:

```
Error: Uncaught NotFoundError: Failed to execute 'setPointerCapture' on 'Element': No active pointer with the given id is found. (test/browser/events.test.js:14011)
        at fireEvent (test/browser/events.test.js:16:6)
        at Context.<anonymous> (test/browser/events.test.js:112:3)
```

I don't quite understand how the pointer is inactive? From reading the spec, it'll implicitly become inactive when `pointerup` or `pointercancel` fire, but neither should be happening here (at least, not before it has a chance to set).

Reply:
Oh, `PointerEvent`s might be their own thing? Funny how you see parts of the docs only after pushing... Will need to try that.

Reply:
Hm, apparently you need to hard-code the pointer ID when using `dispatchEvent`? Weird. Doesn't seem to trigger `got/lostPointerCapture` though.

Reply:
Gave up on actually testing these events, browser behavior is too odd when using `dispatchEvent`. Things don't fire correctly.

Ensuring that `addEventListener` is called with the correct names should be enough anyhow, that's where we were going wrong.